docs/faq.md

```diff
@@ -0,0 +1,46 @@
+
+The dxDAO is a decentralized organization initialized in May of 2019, with over 400 unique stakeholder addresses. With a potential to grow its membership into the thousands, the community-owned and operated organization develops, governs, and promotes DeFi protocols. The dxDAO owns the DutchX trading protocol, and community members are working to launch a prediction market platform (Omen), a state of the art DEX (Mesa), a privacy-centric DeFi dashboard (Mix), and a fundraiser. 
+
+<a name="minimum-order"></a>
+## Why does the Mesa dapp have a warning regarding a minimum amount for orders?  
+Gnosis Protocol is a fully permissionless DEX, in which a centralized operator is replaced by an open, incentivized competition to which anyone can submit order settlement solutions. Those who submit order settlement solutions are referred to as “solvers.” Solvers are economically incentivized to submit order settlement solutions because half of the protocol’s 0.1% fee on trading volume is paid to the solver who submits the best[1] solution (while the other half of the fee is burnt). This fee is automatically included in limit orders on Mesa.
+
+Solvers are expected to act in their own economic self-interest, which means that they will likely optimize for submitting successful solutions that generate higher fee volumes and cover gas costs. **Due to substantially higher gas costs on Ethereum in mid-2020, this means that smaller orders are often not included in solvers’ order settlements. While of course you are guaranteed to never receive below your limit price, this does mean smaller standing orders may sometimes not be filled on the protocol even at market price. Please keep this in mind when placing orders on the protocol; an improvement for this issue is expected for Gnosis Protocol v2.**
+
+While we can warn of this expected behavior for smaller orders, the actual minimum amount for an order to be included in the order settlement solution varies. For example, let's say a solver requires on average 0.1 OWL of fees to be generated by the orders in the solution to be submitted and economically viable. Then, if a single order already contributes 5 OWL of fees, the order settlement solution could theoretically contain any other orders, in particular also very small ones, since the full solution of 30 orders (the maximum amount of orders for any single order settlement solution) is guaranteed to generate more than 30 * 0.1 OWL = 3 OWL of fees. In practice, the average amount of fees a solver optimizes for will likely be determined by the current gas price. It is also possible that other solvers are participating in the open competition with different criteria.
```

Comment:
Is it ok to say `we` in here? `While we can warn ...`, when the title is `Why does the Mesa dapp...`?

Reply:
I will change.